apps/src/dance/Dance.js

```diff
@@ -454,6 +466,31 @@ Dance.prototype.reset = function () {
     getStore().dispatch(showArrowButtons());
     $('#soft-buttons').addClass('soft-buttons-' + softButtonCount);
   }
+  if (this.usesPreview) {
+    this.preview();
+  }
+};
+
+Dance.prototype.preview = async function () {
```

Comment:
Could we add a small comment, maybe covering how this preview features works for newcomers?